src/reports/library/cell.cpp

```diff
@@ -0,0 +1,48 @@
+#include <stdlib.h>
+#include <new>
+#include <iostream>
+#include <stdlib.h>
+
+#include "cell.hpp"
+
+Cell::Cell(int gid, int vgid): m_gid(gid), m_vgid(vgid) {
+    // Compartment id would be gid+compartment_id.
+    //  i.e: for gid 3 and compartment 42 -> 3042
+    // TODO: what should be the id format?
+    m_compartment_id = gid*1000;
```

Comment:
This sounds fragile no ? It assumes that there is always less than 1000 compartments right

Reply:
Yes, this was just a proof of concept, we would have to think in something different